### ISSUE TITLE
Function designator case rule and overloaded operators

### ISSUE THREAD

**Issue by amb5l** [jeremiah-c-leary/vhdl-style-guide#855]
**Environment**
3.13

**Describe the bug**
A function to overload an operator is triggering a `function_017` case violation, but the function name does not contain any letters.

**To Reproduce**
This code in a package body triggers the error:
```
  function "=" (L: std_logic; R: std_logic) return std_logic is
  begin
    return L xnor R;
  end function "=";
```
The tool's suggested solution is `Change ""="" to ""=""`

**Comment by jeremiah-c-leary** [comment]
Morning @amb5l ,

I put in an exception to token case rules if the value is enclosed in double quotes.  I pushed an update to the issue-855 branch.  When you get a chance could you double check it and let me know if it works on your end?

Thanks,

--Jeremy

**Comment by amb5l** [comment]
That works - thanks!

**Comment by jeremiah-c-leary** [comment]
Morning @amb5l,

Awesome, I will merge this to master and then release 3.13.1.

--Jeremy
